packages/carousel/src/ArrowController.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+// import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Icon } from 'pcln-design-system'
```

Comment:
This should probably be imported from `pcln-icons` directly – and, if this is just arrow icons, import the single icons as opposed to the shim for backwards compatibility